web/rootfs/defaults/config.js

```diff
@@ -227,7 +227,7 @@ var config = {
     // minParticipants: 2,
 
     // Use XEP-0215 to fetch STUN and TURN servers.
-    // useStunTurn: true,
+    // useStunTurn: true, //jvb
```

Comment:
In the near future we are going to get rid of this cfg and get it directly from the package, so modifying it is not a great idea. I think we should just use a single setting for now, which would apply to both.

@damencho Do you have an idea on how to better handle this?

Reply:
Hum, probably I will enable that by default in jitsi-meet installation in my PR of adding coturn there by default.

Reply:
now TURN_ENABLE flag change both options